.env

```diff
@@ -23,7 +23,7 @@ _APP_OPENSSL_KEY_V1=your-secret-key
 _APP_DOMAIN=traefik
 _APP_DOMAIN_FUNCTIONS=functions.localhost
 _APP_DOMAIN_SITES=sites.localhost
-_APP_DOMAIN_TARGET=localhost
+_APP_DOMAIN_TARGET=test.appwrite.io
```

Comment:
do we wanna commit this?

Reply:
Yes, we now have proxy tests for rules, and to at least test for 4XX error (not configued DNS) instead of 5xx error (not configured Appwrite instance), we need this to be public domain